### ISSUE TITLE
jekyll gh-pages

### ISSUE THREAD

**Issue by boneskull** [mochajs/mocha#1640]
This is in progress.  If anyone knows how I can use syntax highlighting *and* valid GFM markdown in Jekyll, please let me know.

**Comment by dasilvacontin** [comment]
Awesome! :+1:

**Comment by jbnicolai** [comment]
>  If anyone knows how I can use syntax highlighting and valid GFM markdown in Jekyll, please let me know

@boneskull those should work out of the box with the github-pages gem.

How is progress coming along? Need some help?

**Comment by boneskull** [comment]
yeah.  quite a bit of work has been done here fixing up the documentation.  but I'm obviously misconfiguring something because of the weird markdown I have to use.  let me find my branch

**Comment by boneskull** [comment]
actually, the issue may have been related to the version of liquid in pages-gem.  anyway, I sent a PR and it was merged awhile back, so let me try again.

**Comment by boneskull** [comment]
hey, it's working now

**Comment by jbnicolai** [comment]
:smile:
